webpack/JobWizard/steps/AdvancedFields/DescriptionField.js

```diff
@@ -13,18 +13,18 @@ export const DescriptionField = ({ inputs, value, setValue }) => {
         text: result[0],
       }));
       results.forEach(result => {
-        newDesc = newDesc.replace(
-          result.text,
-          // TODO: Replace with the value of the input from Target Hosts step
-          inputs.find(input => input.name === result.name)?.name || result.text
-        );
+        newDesc = newDesc.replace(result.text, inputValues[result.name] || '');
```

Comment:
The old form left the formatting string untouched if it didn't match any input. If input command was ls and the template was 'Run %{command} %{foo}' then the rendered thing was `Run ls %{foo}`

Reply:
Changed and added tests for it